static/app/utils/issueTypeConfig/index.tsx

```diff
@@ -51,7 +51,8 @@ const BASE_CONFIG: IssueTypeConfig = {
     landingPage: Tab.DETAILS,
     events: {enabled: true},
     openPeriods: {enabled: false},
-    checkIns: {enabled: false},
+    cronChecks: {enabled: false},
+    uptimeChecks: {enabled: false},
```

Comment:
do we need to set every config to be `false` if the `index` one already has it as `false`

Reply:
I guess we can spread the index one, but this makes it a bit more explicit, it doesn't get applied by default since it's a nested object. I can do that here so we don't have to do this again I suppose.

Reply:
ah i see, forgot that these were nested